eventually/team/views.py

```diff
@@ -1,8 +1,12 @@
 """
-views model
-=======================
-
+Views module
+============
 """
-# from django.shortcuts import render
 
-# Create your views here.
+from django.views.generic.base import View
+
+
+class TeamView(View):
+    """Team view handles GET, POST, PUT, DELETE requests.
+    """
+    pass
```

Comment:
when you have docstring you can remove `pass` I think.

Reply:
done